spec/models/validators/Rspec_support_model_helpers_spec.rb

```diff
@@ -0,0 +1,28 @@
+#testing   /spec/support/model_helpers.rb
+require 'rails_helper'
+
+RSpec.describe "Rspec matchers" do
+  let(:test_model) do
+    Class.new do
```

Comment:
As far as I know, it is.
it only exists while the `describe "Rspec matchers"` is running.